src/modules/status.rs

```diff
@@ -3,6 +3,9 @@ use super::{Context, Module, RootModuleConfig};
 use crate::configs::status::StatusConfig;
 use crate::formatter::StringFormatter;
 
+type ExitCode = u32;
```

Comment:
Exit codes can be negative on Windows.
```suggestion
type ExitCode = i64;
```
or
```suggestion
type ExitCode = i32;
```